front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx

```diff
@@ -168,7 +168,13 @@ export default function SpeedSpaceChart({
    * reset chart (only if resetChart is true)
    */
   useEffect(() => {
-    if (resetChart) {
+    if (chart && setSharedXScaleDomain) {
+      setSharedXScaleDomain((prevState) => ({
+        ...prevState,
+        current: prevState.initial,
+        source: undefined,
+      }));
+    } else {
```

Comment:
You also need to keep the if (resetChart)
```suggestion
    if (resetChart) {
      if (chart && setSharedXScaleDomain) {
        setSharedXScaleDomain((prevState) => ({
          ...prevState,
          current: prevState.initial,
          source: undefined,
        }));
      } else {...}
```